src/test/scala/sigmastate/lang/SigmaBinderTest.scala

```diff
@@ -23,10 +23,10 @@ class SigmaBinderTest extends PropSpec with PropertyChecks with Matchers with La
   property("simple expressions") {
     bind(env, "x") shouldBe IntConstant(10)
     bind(env, "b1") shouldBe ByteConstant(1)
-    bind(env, "x+y") shouldBe Plus(10, 11)
+    bind(env, "x-y") shouldBe Minus(10, 11)
```

Comment:
@greenhat  I don't think this is good idea. We still need tests for most often used operation +. The fact that is becomes overloaded makes these tests even more important.